src/test/java/core/basesyntax/CalculationTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.BalanceTransaction;
+import core.basesyntax.strategy.PurchaseTransaction;
+import core.basesyntax.strategy.SupplyTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class CalculationTest extends FruitShopTest {
```

Comment:
I think we must test at least 3 different situations (null input parameters, empty and Ok). Please, add 2 more else tests for this situation